docs/getting-started/customizing-your-setup.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Customizing your setup
+---
+
+# Customizing your setup
+
+Once you've [added your repositories](codacy-quickstart.md), you can customize your setup to adapt the analysis tools to your use case, avoid false positives, and integrate code coverage analysis into your quality pipeline.
```

Comment:
We must be more assertive while recommending to customers that they perform these configurations. Essentially, they'll extract very little value from Codacy if they fail to go through these steps.

Instead of wording the instructions like "you can [...]", let's use imperative or even argumentative language to convince the users of the importance of doing these steps. You can [check the current page for blocking merge requests for examples](https://docs.codacy.com/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate/#adjusting-analysis-settings) on how to do this.

Reply:
Something like this?

```suggestion
Once you've [added your repositories](codacy-quickstart.md), we recommend that you customize your setup to adapt the analysis tools to your use case, avoid false positives, and integrate code coverage analysis into your quality pipeline.
```

Reply:
Fixed